docs/docs/1.2.3/policies/traffic-metrics.md

```diff
@@ -265,17 +269,17 @@ spec:
 
 :::
 ::: tab "Universal"
-This feature requires transparent proxy, therefore for now it's not available in Universal for now.
+This feature requires transparent proxy, so it's currently not available for Universal deployments.
```

Comment:
Actually IIUC there's no correlation between transparent proxy and universal. You can have transparent proxy on universal you just have to set it up yourself.